rs/tests/driver/Cargo.toml

```diff
@@ -130,7 +131,7 @@ serde_bytes = { workspace = true }
 serde_cbor = { workspace = true }
 serde_json = { workspace = true }
 serde_yaml = { workspace = true }
-sha2 = { workspace = true }
+sha2 = "0.10"
```

Comment:
Can't we keep this at the workspace version?

Reply:
Fixed, we should use the same one, my bad